tests/brevitas/quant_tensor/test_quant_tensor.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (C) 2024, Advanced Micro Devices, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+from enum import Enum
+
+import pytest
+import torch
+
+from brevitas.inject.enum import QuantType
+from brevitas.nn import QuantIdentity
+from brevitas.quant_tensor import QuantTensor
+
+
+class Operator(Enum):
+    ADD = 0
+    SUBTRACT = 1
+    DIVIDE = 2
+    MULTIPLY = 3
+    MATMUL = 4
+
+
+def to_quant_tensor(input: torch.Tensor) -> QuantTensor:
+    mod = QuantIdentity(bit_width=8, return_quant_tensor=True)
+    return mod(input)
+
+
+def qdq(normal_tensor, quant_tensor):
+    return (
+        torch.round(normal_tensor / quant_tensor.scale + quant_tensor.zero_point) -
+        quant_tensor.zero_point) * quant_tensor.scale
+
+
+def test_quant_tensor_init():
+    x = torch.randn(4, 4)
+    quant_tensor = to_quant_tensor(x)
+    normal_tensor = torch.Tensor(x)
+    assert torch.allclose(qdq(normal_tensor, quant_tensor), quant_tensor, rtol=0.01)
```

Comment:
difference between the qdq result and quant tensor is extremely close but some error is creeping in from the quanttensor somewhere so added relative tolerance